tools/tensorflow/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.11
+RUN apt-get update
+RUN apt-get install -y build-essential
+RUN apt-get install -y libhdf5-dev
```

Comment:
In general you want to put `apt-get update` and `apt-get install` commands in the same `RUN`; take a look at this doc page for more explanation about why: https://docs.docker.com/build/building/best-practices/#apt-get